sorc/chgres_cube.fd/input_data.F90

```diff
@@ -27,7 +27,7 @@ module input_data
                                     convert_nst, &
                                     orog_dir_input_grid, &
                                     orog_files_input_grid, &
-                                    tracers_input, num_tracers, &
+                                    tracers_input, num_tracers_input, &
```

Comment:
@HenryJuang-NOAA Why did you change 'num_tracers' to 'num_tracers_input' here and in other places?  Was there a bug in chgres?

Reply:
Yes, it is a bug for my case.  Since in my condition, input tracer number and output tracer number are different. So I need tracer number for input.

Reply:
There are situations when the the number of input and output tracers can be different. So I need to think about what you are doing some more.

Reply:
> There are situations when the the number of input and output tracers can be different. So I need to think about what you are doing some more.

okay.  I need ti change it otherwise dimension wrong in some code for tracer arrays...

Reply:
Ok, I see what you are doing.